POD in backup-manager-upload has been cleaned.
This patch has been applied upstream.

Provided by: Alexis Sukrieh <sukria@sukria.net>

---
 backup-manager-upload |    5 +++++
 1 file changed, 5 insertions(+)

--- a/backup-manager-upload
+++ b/backup-manager-upload
@@ -37,9 +37,12 @@
 
 Select the user to use for connection.
 
+=back 
 
 =head1 OPTIONAL ARGS
 
+=over 4
+
 =item B<--password=>I<password>
 
 Select the ftp user's password (only needed for ftp transfers).
@@ -94,6 +97,8 @@
 
 Date pattern to select some files to upload, can be a valid date (YYYYMMDD) or 'today' or 'yesterday'.
 
+=back
+
 =head1 ERROR CODES
 
 If something goes wrong during an upload, backup-manager-upload will exit 
